Loop.xcodeproj/project.pbxproj

```diff
@@ -1448,6 +1450,7 @@
 			files = (
 				43F5C2C91B929C09003EB13D /* HealthKit.framework in Frameworks */,
 				1DC63E7425351BDF004605DA /* TrueTime.framework in Frameworks */,
+				C19F496325630504003632D7 /* Minizip.framework in Frameworks */,
```

Comment:
[nit] This is a strange edit to the project file.  Intentional?

Reply:
I've had to add the Minizip framework to satisfy missing symbols like `_zipClose`, `_zipOpen`, etc, several times now on the dev branch; I'm surprised it hasn't been seen by others?